### ISSUE TITLE
test failure on centos7

### ISSUE THREAD

**Issue by jscook2345** [apptainer/singularity#3700]
## Version of Singularity:

master

```
--- FAIL: TestConfigure (0.00s)
    --- FAIL: TestConfigure/with_unconfined_SELinux_context (0.00s)
        security_test.go:126: unexpected failure with unconfined SELinux context: write /proc/self/task/20449/attr
/exec: invalid argument
```

@cclerget 

Ideas on why this is failing on centos7?

**Comment by cclerget** [comment]
@jscook2345 Is `unconfined` module loaded ? `semodule -l` ?

**Comment by jscook2345** [comment]
@cclerget 

```
[vagrant@localhost singularity]$ sudo semodule -l | grep uncon
unconfined      3.5.0
unconfineduser  1.0.0
```

**Comment by cclerget** [comment]
@jscook2345 Should be fixed by #3723
